application.properties

```diff
@@ -0,0 +1,10 @@
+server.port = 8081
+spring.thymeleaf.cache=false
```

Comment:
remove the thymeleaf, application and datasource variables. 
They are not necessary :)